chpl/chpl-service/src/main/java/gov/healthit/chpl/util/ValidationUtils.java

```diff
@@ -17,21 +18,31 @@
 
     public static boolean hasNewline(String input) {
         //check both windows and unix line separator chars
-        if(input == null || StringUtils.isEmpty(input)) {
+        if (input == null || StringUtils.isEmpty(input)) {
             return false;
         }
         return input.contains("\n") || input.contains("\r\n");
     }
-    
+
+    /**
+     * Validate that input string is a well formed URL.
+     * @param input URL to check
+     * @return true iff input is a valid URL. Will return false for empty/null string
+     */
+    public static boolean isWellFormedUrl(final String input) {
+        UrlValidator validator = new UrlValidator();
```

Comment:
Can UrlValidator be a static member of this class so that it only gets created once? Or does it need to be created each time this method is called?